.github/workflows/release.yml

```
@@ -2,96 +2,38 @@ name: Release
 on:
   push:
     tags:
-      - '[2-9].[0-9]+.[0-9]+'
-      - '[2-9].[0-9]+.[0-9]+-*'
+      - '[0-9].[0-9]+.[0-9]+'
+      - '[0-9].[0-9]+.[0-9]+-*'
 env:
   DOTNET_NOLOGO: true
 jobs:
   release:
-    runs-on: windows-2019 # Code signing requirement https://github.com/NuGet/Home/issues/7939
+    runs-on: ubuntu-20.04 # Code signing requirement https://github.com/NuGet/Home/issues/7939
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

Explicit versions on all actions. These only got that way because of a bug in Dependabot that got fixed. If we put them back to specific versions we'll get proper Dependabot PRs again.
```suggestion
        uses: actions/checkout@v3.0.2
```